### PR TITLE
Exploration

### PR DESCRIPTION
**Pull request by manthan99**
This PR adds the pure pursuit path tracker required for running gbplanner2

### CODE REVIEW

File: smb_path_tracker/package.xml
Comment (mantelt) [inline smb_path_tracker/package.xml:5]: Could you add you (or@nospelt?) as maintainer and define the original author as author? 
Somehow, this seems like plagiarism to me.

**Review by mantelt — CHANGES_REQUESTED** [review]
Looks ok to me, except for the license. I would feel better if it would be clear where the original one is from and who wrote it.

File: smb_path_tracker/launch/pure_pursuit_tracker.launch
Comment (mantelt) [inline smb_path_tracker/launch/pure_pursuit_tracker.launch:4]: we have a muxer that "fuses" different topics with twist commands depending on their priority. 
So this tracker should publish to its own topic (e.g. `/control/pure_pursuit/cmd_vel`) and the [muxer adjusted accordingly](https://github.com/ETHZ-RobotX/smb_common/blob/devel/smb_control/config/twist_mux.yaml).

**Review by mantelt — CHANGES_REQUESTED** [review]
Sorry, found some other minor thing that I would change for the sake of consistency.

File: smb_path_tracker/launch/pure_pursuit_tracker.launch
Comment (manthan99) [reply smb_path_tracker/launch/pure_pursuit_tracker.launch]: Thanks for the suggestion. I have now changed it

**Comment by manthan99** [comment]
Hi @mantelt 

As discussed, I have now created a new package called as smb_exploration which consists of all the relevant launch files and config files for both the exploration planners. Moroever, I have also added some basic documentation to build and run the exploration planners.

Let me know once you have had a look and we can then merge it into devel. Though it is not urgent

**Review by mantelt — CHANGES_REQUESTED** [review]
Please quickly add some info about the origin of the pure pursuit package into the readme.md. Otherwise, I think we can merge.

**Comment by manthan99** [comment]
So I have made the changes as requested and the PR is good to merge.

We can merge it into devel once Shobhit's PR (https://github.com/ETHZ-RobotX/smb_common/pull/42) is merged into exploration

**Comment by manthan99** [comment]
I will quickly test this branch now and the full TF tree with MSF graph. If it goes well, I will merge this into devel

**Pull request merged**
